### PR TITLE
gubernator: handle missing index when doing github xref lookups

### LINKED ISSUE #10920: Gubernator returns no matching index error
**What happened**:
I have just deployed Gubernator on my GCP project but got the below internal error when access the test result.

```
Traceback (most recent call last):
  File "/base/alloc/tmpfs/dynamic_runtimes/python27g/f612b68945952bd1/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 1535, in __call__
    rv = self.handle_exception(request, response, e)

NeedIndexError: no matching index found.
The suggested index for this query is:
- kind: GHIssueDigest
  properties:
  - name: xref
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Please provide links to example occurrences, if any**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by Katharine**
If you deploy Gubernator without any GitHub integration, `GHIssueDigests` are never created and the datastore queries for them fail. While we don't expect any GitHub-related functionality to work in this setup, it would be nice if the basic GCS reading didn't explode.

Handle the exceptions thrown. This should be a no-op for us but helpful to others.

Fixes #10920 (hopefully; it's hard to test)

/cc @ibzib

### CODE REVIEW

File: gubernator/github/models.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import json
 
 import google.appengine.ext.ndb as ndb
+import google.appengine.api.datastore_errors as datastore_errors
```
Comment (fejta) [inline gubernator/github/models.py]: `from google.appengine.api import datastore_errors` is the preferred style: http://google.github.io/styleguide/pyguide.html#224-decision

**Comment by BenTheElder** [comment]
bump

**Comment by fejta** [comment]
/test all
Do we still want this?

**Pull request closed without merging**